internal/render/worker/statefulset.go

```diff
@@ -110,3 +103,21 @@ func RenderStatefulSet(
 		},
 	}, nil
 }
+
+// BuildAnnotations returns a map of annotations for the Slurm worker pods
+func BuildAnnotations(clusterName string) map[string]string {
+	annotations := map[string]string{
+		consts.DefaultContainerAnnotationName: consts.ContainerNameSlurmd,
+	}
+
+	// Always set the annotation for ContainerNameMunge to unconfined
```

Comment:
May be lets do that for all containers and remove that comment?

Reply:
it has already unconfined for all containers except Slurmd in apapparmor profile case